zebra-network/src/protocol/external/codec/tests.rs

```diff
@@ -0,0 +1,366 @@
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
```

Comment:
Nitpicks:
- We usually put fixed test vectors in a `tests/vectors.rs` module
- Please add a short module doc comment

Reply:
Fixed in [`10c405c`](https://github.com/ZcashFoundation/zebra/pull/5835/commits/10c405ca251398f016e76909ecc970294bcb43d6).